apps/testnorge-arena/arena-core/src/main/java/no/nav/registre/arena/core/service/RettighetTiltakService.java

```diff
@@ -163,9 +174,115 @@
                 }
             }
         }
+
+        var identerMedOpprettedeEndreDeltakerstatus = endreDeltakerstatus(
+                responses,
+                rettigheter,
+                miljoe);
+
+        if(!identerMedOpprettedeEndreDeltakerstatus.isEmpty()){
+            for (String ident : responses.keySet()) {
+                if(identerMedOpprettedeEndreDeltakerstatus.get(ident) != null) {
+                    for (var status : identerMedOpprettedeEndreDeltakerstatus.get(ident)) {
+                        responses.get(ident).get(0).getFeiledeRettigheter().addAll(
+                                status.getFeiledeRettigheter());
```

Comment:
Denne stusser jeg litt over... Legger du til feilmeldingen på en tidligere respons? Gir det ikke mer mening å istedenfor å ha:
responses.get(ident).get(0).getFeiledeRettigheter().addAll(status.getFeiledeRettigheter());

så har du:
responses.get(ident).add(status);

eller misforstår jeg?

Reply:
Ja, jeg har oppdatert den litt nå. Det jeg gjør er å legge til response for endre deltakerstatus til response for opprett tiltaksdeltakelse.

Reply:
Ah, skjønner, da gir det mening!